docs/_config.yml

```diff
@@ -0,0 +1,62 @@
+title                       : Quansight Labs Jupyter CZI Accessibility Grant  
+author                      : Quansight Labs
+copyright                   : "2021"  
+logo                        : ""  
```

Comment:
I seem to have remembered seeing a discussion about Jupyter a11y logos.... can we do some digging and see if that landed somewhere and maybe adopt a logo?